### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ruwhan**
add http://frontenddeveloperjob.com/

### CODE REVIEW

**Comment by lukasz-madon** [comment]
not sure how to filter

**Comment by ruwhan** [comment]
@lukasz-madon it is mentioned in the readme, do sort by `performed`.

**Comment by waldyrious** [comment]
To clarify, you need to change the display from cards to table (icons at the top left of the listing) before you can sort by the "performed" column. That should probably be included in the instructions.

**Comment by ruwhan** [comment]
thanks @waldyrious , yes you are right, forgot that because it is stored in cookies and never change that again (only done that once). @lukasz-madon please review again.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ A curated list of awesome [remote working](http://en.wikipedia.org/wiki/Telecomm
   1. [Remote Hackers](http://remotehackers.com/)
   1. [weworkremotely.com](https://weworkremotely.com/)
   1. [WFH.io](https://www.wfh.io/)
+  1. [Front-end Developer Jobs] (http://frontenddeveloperjob.com/) View as table, then sort by `performed`
```
Comment (waldyrious) [inline README.md]: I'd suggest the phrasing "change to table view" instead.

**Pull request merged**